### PR TITLE
Abort the watcher if stdin is closed to avoid zombie processes

### PR DESCRIPTION
**Pull request by josevalim**
This is the same behaviour as found in postcss --watch:

https://github.com/postcss/postcss-cli/blob/f0e262ed484436a669d81b66424ef6017058950c/index.js#L55-L56

### CODE REVIEW

**Comment by josevalim** [comment]
You can test this by running `npx tailwindcss --watch` and then hitting Ctrl+D.

File: src/cli.js [hunk 1]
```diff
@@ -784,6 +784,9 @@ async function build() {
   }
 
   if (shouldWatch) {
+    /* Abort the watcher if stdin is closed to avoid zombie processes */
+    process.stdin.on('end', () => process.exit(0));
+    process.stdin.resume();
```
Comment (josevalim) [inline src/cli.js:789]:
```suggestion
    process.stdin.resume()
```

File: src/cli.js [hunk 2]
```diff
@@ -784,6 +784,9 @@ async function build() {
   }
 
   if (shouldWatch) {
+    /* Abort the watcher if stdin is closed to avoid zombie processes */
+    process.stdin.on('end', () => process.exit(0));
```
Comment (josevalim) [inline src/cli.js:788]:
```suggestion
    process.stdin.on('end', () => process.exit(0))
```

**Comment by adamwathan** [comment]
Thank you! I'll get a patch out in the next day or so with this change 👍🏻

**Pull request merged**